docs/INSTALL.md

```diff
@@ -0,0 +1,635 @@
+
+; choose a non-zero value for session.gc_probability, otherwise old session data will not be deleted
+session.gc_probability = 1
+session.gc_divisor = 100
+session.gc_maxlifetime = 14400
+session.hash_function = 0
+ 
+; for chat server since ILIAS 4.2
+allow_url_fopen = 1
+
+; How many GET/POST/COOKIE input variables may be accepted
+max_input_vars = 10000
+```
+
+Remember to reload your web server configuration to apply those changes.
+
+<a name="database-installationconfiguration"></a>
+## Database Installation/Configuration
+
+On Debian/Ubuntu execute: 
+```
+apt-get install mysql-server
+```
+
+On RHEL/CentOS execute: 
+```
+yum install mariadb
+```
+
+We RECOMMEND to create a dedicated database user for ILIAS:
+
+```
+mysql -u root -p
+CREATE DATABASE ilias CHARACTER SET utf8 COLLATE utf8_general_ci;
+CREATE USER 'ilias'@'localhost' IDENTIFIED BY 'password';
+GRANT LOCK TABLES on *.* TO 'ilias@localhost';
+GRANT ALL PRIVILEGES ON ilias.* TO 'ilias'@'localhost';
+FLUSH PRIVILEGES;
+```
+
+<a name="mysql-perfomance-tuning-optional"></a>
+### MySQL Perfomance tuning (OPTIONAL)
+
+We RECOMMEND to use https://github.com/major/MySQLTuner-perl to optimize your MySQL configuration (e.g. ```/etc/mysql/my.cnf```). Execute ```mysqltuner.pl``` after several days of using ILIAS in production.
+
+<a name="e-mail-configuration-optional"></a>
+## E-Mail Configuration (OPTIONAL)
+
+You MAY use whatever MTA you like to send E-Mail generated by ILIAS. We RECOMMEND to use an already existing smarthost (mailhub). A very simple way to do so is using ```ssmtp```:
+
+On Debian/Ubuntu execute: 
+```
+apt-get install ssmtp
+```
+
+On RHEL/CentOS execute: 
+```
+yum install ssmtp
+```
+
+The configuration file for SSMTP (e.g. ```/etc/ssmtp/ssmtp.conf ```) MAY look as follows:
+
+```
+#
+# Config file for sSMTP sendmail
+#
+# The person who gets all mail for userids < 1000
+# Make this empty to disable rewriting.
+root=yourmail@mail.com
+
+# The place where the mail goes. The actual machine name is required no 
+# MX records are consulted. Commonly mailhosts are named mail.domain.com
+mailhub=smtp.yourmail.com
+
+# Where will the mail seem to come from?
+rewriteDomain=yourservername.org
+
+# The full hostname
+hostname=yourserver.example.com
+
+# Are users allowed to set their own From: address?
+# YES - Allow the user to specify their own From: address
+# NO - Use the system generated From: address
+FromLineOverride=YES
+```
+
+<a name="install-other-depedencies"></a>
+## Install other Depedencies
+
+On Debian/Ubuntu execute: 
+```
+apt-get install zip unzip php5-gd php5-mysql php5-xslt imagemagick openjdk-7-jdk
+```
+
+On RHEL/CentOS execute: 
+```
+yum install zip unzip php-gd libxslt ImageMagick java-1.7.0-openjdk
+```
+<a name="installation-wizard"></a>
+## Installation Wizard
+
+After having all dependencies installed an configured you should be able to run the ILIAS Installation Wizard using http://yourservername.org/setup/setup.php
+
```

Comment:
After having all dependencies installed an# d configured, restart the apache2 server in order for the packages to take effect. Then you should be able to run.....

If apache is not reloaded there are to unmet dependencies in the setup. The user installing ILIAS does not get a well enough error message to pinpoint the problem. Although XLS and GD are both installed ILIAS does not see them.